### PR TITLE
Allow oc-mirror-e2e-unit to be within build02

### PR DESCRIPTION
**Pull request by sfxworks**
Relates to https://coreos.slack.com/archives/CBN38N3MW/p1639081903350000

### CODE REVIEW

File: core-services/sanitize-prow-jobs/_config.yaml [hunk 1]
```diff
@@ -5383,6 +5383,8 @@ groups:
     - openshift-release-infra-periodics.yaml
     - infra-periodics-origin-release-images.yaml
   build02:
+    jobs:
+    - oc-mirror-e2e-unit
```
Comment (hongkailiu) [inline core-services/sanitize-prow-jobs/_config.yaml:5387]: Could you please name the job as `org-repo-branch-testname`?

Comment (hongkailiu) [reply core-services/sanitize-prow-jobs/_config.yaml:5387]: Actually, can you do this together with https://github.com/openshift/release/pull/24134/files ?

Comment (sfxworks) [reply core-services/sanitize-prow-jobs/_config.yaml:5387]: Sure! I'll do this today. Thanks!

Comment (sfxworks) [reply core-services/sanitize-prow-jobs/_config.yaml:5387]: I have combined the two into https://github.com/openshift/release/pull/24492

**Comment by sfxworks** [comment]
Closed in favor of https://github.com/openshift/release/pull/24492

**Pull request closed without merging**